app/views/branding_groups/list_navigate.html.erb

```diff
@@ -0,0 +1,44 @@
+<h2><%= link_to 'Home', site_path %> / Collections</h2>
+<span>Collections are curated groups of studies which share a common research theme. If your organization is interested in creating a collection within SCP, contact us at <%= mail_to 'scp-support@broadinstitute.zendesk.com', 'scp-support@broadinstitute.zendesk.com' %></span>
+<div class="row">
+  <div class="col-md-12">
+    <ul class="collections-list">
+      <% @branding_groups.each do |branding_group| %>
+        <li id="<%= branding_group.name_as_id %>">
+          <div class="collections-logo">
+            <% if branding_group.splash_image.filename.present? %>
+              <%= link_to image_tag(branding_group.splash_image.url), site_path(scpbr: branding_group.name_as_id) %>
```

Comment:
Non-blocking: it would be nice if we had better behavior for non-existent images as opposed to the default "broken image tag" for most browsers.

Reply:
yup, that was addressed in an earlier PR, hence the `if` above.  If there's no image, it will just be empty

Reply:
Ah, that means then that I have a bunch of broken images in my local instance.  Good to know!